### PR TITLE
base-image-to-ubuntu-16-04

### PR DESCRIPTION
**Pull request by kelepirci**
Since Ubuntu 14:04 LTS has come to EOL, new base image should be Ubuntu 16.04 LTS.

### CODE REVIEW

**Comment by kelepirci** [comment]
- Changed APACHEDS_VERSION 2.0.0.AM25
- Updated base image to ubuntu-16.04
- Tested locally

**Comment by joshmoore** [comment]
Thanks, @kelepirci. Targeting this for 0.7.0.

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,8 @@ VOLUME ${APACHEDS_DATA}
 
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections \
     && apt-get update \
-    && apt-get install -y ldap-utils procps openjdk-7-jre-headless curl jq \
+    && apt-get install -y apt-utils \
+    && apt-get install -y ldap-utils procps openjdk-8-jre-headless curl jq \
```
Comment (manics) [inline Dockerfile]: Would you mind rewriting this so it'll be easier to make future changes:
```
&& apt-get install -y \
    apt-utils \
    ldap-utils \
    ...
```

Comment (kelepirci) [reply Dockerfile]: Fixed

**Review by manics — APPROVED** [review]
Thanks!

**Comment by joshmoore** [comment]
Pushed the 0.7.0 tag. Docker Hub should start building shortly. Thanks, both!

**Pull request merged**